birdhouse/components/weaver/config/weaver/weaver.ini

```diff
@@ -0,0 +1,157 @@
+# Weaver settings configuration
```

Comment:
In fact you've probably committed all the "generated" files from their "template" origin.